server/service/integration_mdm_test.go

```diff
@@ -3078,18 +3078,7 @@ func (s *integrationMDMTestSuite) TestTeamsMDMAppleDiskEncryption() {
 	// retrieving the team returns the disk encryption setting
 	var teamResp getTeamResponse
 	s.DoJSON("GET", fmt.Sprintf("/api/latest/fleet/teams/%d", team.ID), nil, http.StatusOK, &teamResp)
-	require.True(t, teamResp.Team.Config.MDM.MacOSSettings.EnableDiskEncryption)
-
-	// apply with invalid disk encryption value should fail
```

Comment:
Isn't this test possible anymore ?

Reply:
you're right! I think at some point I assumed it wasn't necessary, but we still support the old config. I added it back